types/modal.d.ts

```diff
@@ -128,7 +128,7 @@ export interface ModalOptions {
    */
   onOk?: () => any;
 
-  parentContext: Object;
+  parentContext?: Object;
```

Comment:
It might also be incorrect to use `Object` as the type here. This is officially discouraged by TypeScript: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

The non-primitive `object` type is preferred - assuming there isn't a more specific type that could be assigned here?

Reply:
you are right.    Just do it